recipes/boost/all/conanfile.py

```diff
@@ -132,6 +134,7 @@ def package_id(self):
             self.info.header_only()
             self.info.options.header_only = True
         else:
+            del self.info.options.pch
```

Comment:
Thanks, this is saving me writing a PR myself today.

Does the pch option affect the build only? Just double-checking, because I personally don't know enough to know if I need to pull it from options.

Reply:
as I understood from the docs, it's only to speed up the compilation

Reply:
I am wondering if it makes sense as a recipe option or if it could be "hardcoded" as it is just to compile faster. But let's continue with this as is just in case someone wants to opt-out